.github/workflows/cron-streams-active-cleanup.yaml

```diff
@@ -0,0 +1,15 @@
+name: "cron: Active streams cleanup"
```

Comment:
I wonder if we shouldn't run it as a kubernetes Cron Jobs. I know that currently we don't have a clear agreement on whether cron jobs should be GH Actions or Kubernetes CronJobs, but the advantages I see for integrating it in our Infra:
- Alerts integration (Grafana Pager Duty)
- (super minor) GH Actions load on the GH workers
- (minor) security and passing admin LP token

Anyway, I leave it up to you.

Reply:
Even though I agree it would be much better to run cronjobs, I'd leave that as a separate change for the future, given we have other (more critical) workflows already running on GitHub actions, mainly usage billing. We should probably bring this up as a tech debt to be prioritized eventually though.